lib/api/2.0/nodes.js

```diff
@@ -61,24 +63,55 @@ var nodesGetPollersById = controller(function(req) {
 });
 
 var nodesGetWorkflowById = controller(function(req) {
-    return nodes.getNodeWorkflowById(req.swagger.params.identifier.value);
+    var newQuery;
+    var swaggerQuery = req.swagger.query || {};
+    if (_(swaggerQuery).has('active')) {
+        if (swaggerQuery.active) {
+            newQuery = ({
+                _status: constants.Task.ActiveStates
+            });
+        } else {
+            newQuery = ({
+                _status: {'!': constants.Task.ActiveStates}
+            });
+        }
+        delete swaggerQuery.active;
+        newQuery = _.merge({}, swaggerQuery, newQuery, req.query);
```

Comment:
@BillyAbildgaard swaggerQuery shouldn't be included here because can include parameters that don't directly map to workflow model properties.